packages/data-stores/src/plans/resolvers.ts

```diff
@@ -79,13 +79,13 @@ function calculateDiscounts( planProducts: PlanProduct[] ) {
 	}
 }
 
-function processFeatures( features: Feature[] ) {
+function processFeatures( features: DetailsAPIFeature[] ) {
 	return features.reduce( ( features, feature ) => {
 		features[ feature.id ] = {
 			id: feature.id,
 			name: feature.name,
 			description: feature.description,
-			type: feature.type ?? 'checkbox',
+			type: 'checkbox',
```

Comment:
For future reference: If the feature's type is always a checkbox, what's the added value? Do we still have to include it?

Reply:
Seems like this is not needed, but we should investigare further in a separate issue